src/rt/sections_elf_shared.d

```diff
@@ -274,6 +274,8 @@ else
      */
     Array!(void[])* initTLSRanges() nothrow @nogc
     {
+        foreach (ref pdso; _loadedDSOs)
```

Comment:
Before this, I'll add `safeAssert (_tlsRanges.empty);` to make sure that it's only initialized once.